vmdb/db/migrate/20150311181430_add_draft_to_orchestration_templates.rb

```diff
@@ -0,0 +1,5 @@
+class AddDraftToOrchestrationTemplates < ActiveRecord::Migration
+  def change
+    add_column :orchestration_templates, :draft, :boolean, :default => false
```

Comment:
@Fryguy Should we be setting the default through `default_value_for` or is it ok here?